src/ngraph/pass/fused_op_decomposition.hpp

```diff
@@ -25,6 +25,30 @@ namespace ngraph
 {
     namespace pass
     {
+        /// \brief  The FusedOpDecomposition pass is used to decompose a fused op
+        /// if it is not supported by the backend.
+        ///
+        /// \details The pass is run on each node in the computation graph. If the
+        /// node is a fused op and is not supported by the backend, the pass
+        /// decomposes it and is run recusrsively until no more fused ops
+        /// can be found or the new ops are supported by the backend.
+        /// It also adds provenance tags along the way to each op for easy reference
+        /// and debugging.
+        /// The backend can optionally provide a callback function to indicate if a
+        /// fused op is supported or not while registering the pass.
+        ///
+        /// In the example shown below, the original graph has a fused Gelu op.
```

Comment:
GeLU if this is being documented as a comment, use acronym-sensitive capitalization.